docs/docs/create-redwood-app.md

```diff
@@ -0,0 +1,100 @@
+---
+slug: create-redwood-app
+description:
+---
+
+# Create Redwood App
+
+To get up and running with Redwood, you can use the Create Redwood App:
+
+```terminal
+yarn create redwood-app <your-app-name>
+```
+
+## Set up for success
+Redwood requires that you're running Node version 18.0.0 or higher.
+
+If you're running Node version 19.0.0 or above, you can still use Create Redwood app, but it may make your project incompatible with some deploy targets, such as AWS Lambdas.
```

Comment:
Also just to stay consistent, I don't think we have to vary it yet it doesn't sound too tinny

```suggestion
If you're running Node version 19.0.0 or higher, you can still use Create Redwood app, but it may make your project incompatible with some deploy targets, such as AWS Lambdas.
```